core/src/main/java/org/apache/carbondata/core/datastore/page/DecimalColumnPage.java

```diff
@@ -106,4 +109,48 @@ public void setDoublePage(double[] doubleData) {
     throw new UnsupportedOperationException("invalid data type: " + dataType);
   }
 
+  private BigDecimal getDecimalFromRawData(int rowId) {
```

Comment:
Please add comment to describe when to use this method and the below method

Reply:
Fixed